server/komodo-rest/src/main/java/org/komodo/rest/TeiidSwarmMetadataInstance.java

```diff
@@ -78,6 +78,8 @@ public TeiidSwarmMetadataInstance(TeiidConnectionProvider connectionProvider) {
 							scd.setType(type.name());
 							ServiceBinding binding = getServiceBinding(svc);
 							if (binding == null) {
+								scd.setBound(false);
+							} else {
 								scd.setBound(true);
```

Comment:
Does this not change this logic to the opposite. Is that expected?

Reply:
yes, that was bug. I also have in my changes.

Reply:
The original logic did not look correct.  Seems that if the binding is not found, then 'bound' should be false?